app/app.ts

```diff
@@ -49,4 +53,10 @@ export class DoseAmigosApp {
 
 }
 
-ionicBootstrap(DoseAmigosApp);
+ionicBootstrap(
+    DoseAmigosApp,
+    [], /* custom providers */
+    {
+        mode: "md"
```

Comment:
This tells the application to use the Material Design UI on all devices, instead of trying to match ios or windows phone themes.